lib/index.js

```diff
@@ -17271,1303 +17271,1303 @@ exports.parseURL = __nccwpck_require__(2158).parseURL;
 /***/ ((module, __unused_webpack_exports, __nccwpck_require__) => {
```

Comment:
GPT summary of [f9afda](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/c8c47408c482f791430fa4be8b1253005844ef70/lib/index.js#f9afdaf99a700608749a1c13d0a44e17cf9d0b8e) - [9dfabe](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/14bc7d58520d55fecaa099a6e45930d84b7be731/lib/index.js#9dfabe87b6a1e83bdef5cab131d396d6c40293f3):
Error: couldn't generate summary